config/routes.rb

```diff
@@ -38,11 +38,22 @@
             post 'sync', to: 'studies#sync_study'
           end
           get 'explore', to: 'visualization/explore#show'
+          get 'explore/cluster_options', to: 'visualization/explore#cluster_options'
+
           resources :expression, controller: 'visualization/expression', only: [:show], param: :data_type
```

Comment:
I would prepend all of this with `namespace :visualization`, and then we can remove the `controller:` parameter as the controller inheritance will match (e.g. `Api::V1::Visualization::ExpressionController` in `app/controllers/api/v1/visualization/expression_controller.rb`).